setup.cfg

```diff
@@ -12,7 +12,7 @@ upload-dir = docs/build/html
 [flake8]
 max-line-length = 99
 ignore = W503, W605
-extend-ignore = E203
+extend-ignore = E203, E704
```

Comment:
what kind of error is this?

Reply:
having multiple statements in one line. got triggered cause black likes to put the Ellipsis at the same line as an overloaded function definition